ansible_collections/arista/avd/molecule/eos_validate_state/intended/test_catalogs/dc1-leaf1a-catalog.yml

```diff
@@ -496,6 +496,13 @@ anta.tests.routing.bgp:
       - 10.255.0.2
     result_overwrite:
       custom_field: 'BGP EVPN Peer: dc1-spine2 (IP: 10.255.0.2)'
+- VerifyBGPSpecificPeers:
+    address_families:
+    - afi: evpn
+      peers:
+      - 10.1.1.1
+    result_overwrite:
+      custom_field: 'BGP EVPN Peer: (IP: 10.1.1.1)'
```

Comment:
Should we have something like `BGP EVPN Peer: 10.1.1.1` instead you think? Only for the use case where we don't have the peer.